src/libmain/shared.cc

```diff
@@ -363,6 +363,7 @@ void printVersion(const std::string & programName)
             << "\n";
         std::cout << "Store directory: " << settings.nixStore << "\n";
         std::cout << "State directory: " << settings.nixStateDir << "\n";
+        std::cout << "Data directory: " << settings.nixDataDir << "\n";
```

Comment:
Is that related to the rest?

Reply:
Not really related, it was just useful during debugging this issue.